test/integration/tcp_proxy_integration_test.cc

```diff
@@ -84,15 +72,6 @@ TEST_P(TcpProxyIntegrationTest, SendTlsToTlsListener) {
         context = Ssl::createClientSslContext(false, false, *context_manager);
         ssl_client = dispatcher_->createSslClientConnection(*context, address);
       },
-      // Set up the initial REST response for the ssl_auth filter to avoid the async client doing
-      // reconnects.  Loopback is whitelisted by default so no response payload is necessary.
-      [&]() -> void {
-        fake_rest_connection = fake_upstreams_[1]->waitForHttpConnection(*dispatcher_);
-        request = fake_rest_connection->waitForNewStream();
-        request->waitForEndStream(*dispatcher_);
-        request->encodeHeaders(Http::TestHeaderMapImpl{{":status", "200"}}, false);
-        request->encodeData(0, true);
-      },
```

Comment:
Is this the conflict you meant?
If so please feel free to check in ASAP, because it is way cleaner and I'm happy to do the merges.